### PR TITLE
cli: add an option timeout to control the request

### PR DESCRIPTION
**Pull request by menduo**
Thank you for contributing to sewer.                    
Every contribution to sewer is important to us.                       
You may not know it, but you have just contributed to making the world a more safer and secure place.                         

Contributor offers to license certain software (a “Contribution” or multiple
“Contributions”) to sewer, and sewer agrees to accept said Contributions,
under the terms of the MIT License.
Contributor understands and agrees that sewer shall have the irrevocable and perpetual right to make
and distribute copies of any Contribution, as well as to create and distribute collective works and
derivative works of any Contribution, under the MIT License.

Now,                   

## What(What have you changed?)

cli: add an option timeout to control the request

## Why(Why did you change it?)

In some area, the way to ACME server is not very good.

### CODE REVIEW

File: sewer/dns_providers/tests/test_aliyundns.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import logging
```
Comment (komuw) [inline sewer/dns_providers/tests/test_aliyundns.py]: Codacy Issue found: [Unused import logging](https://app.codacy.com/manual/komuW/sewer/pullRequest?prid=5237189)

**Comment by komuw** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/komuW/sewer/pullRequest?prid=5237189&bid=17166219)

**Comment by mmaney** [comment]
The timeout change is all in the one commit 2fba8a32, right?  Could you make a clean PR by taking a clean clone of komuw's master branch and then adding the timeout changes on top?  That looks good, but I'd have to -1 on the PR as it stands.

File: sewer/dns_providers/tests/test_aliyundns.py (hunk 1 above)
Comment (menduo) [reply sewer/dns_providers/tests/test_aliyundns.py]: ok, sorry, fixed.

**Comment by mmaney** [comment]
@menduo - my bad, I was looking over the PRs and bugs too late in the evening and confused this one with some other timeout issues that are addressed in #187.  This specific change to allow control from sewer-cli is NOT in the current version of the PR, perhaps because I deferred most or all of the cli.py updates until the dust settled.  Reopening this to remind myself, though I think it's already on my todo list.

**Comment by mmaney** [comment]
How strange, this jumped to the top of the todo list on its own.  Reworked as #188, now this is done.

**Comment by menduo** [comment]
@mmaney Ok, great!

**Pull request closed without merging**